src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -534,44 +567,37 @@ def query_example_transactions(
     if per_suspect == 0:
         return {}
 
-    orderby_columns = SPAN_PERFORMANCE_COLUMNS[orderby].suspect_example_sort
-
     selected_columns: List[str] = [
         "id",
         "project.id",
-        "array_join(spans_op)",
-        "array_join(spans_group)",
-        *orderby_columns,
     ]
 
-    builder = QueryBuilder(
+    builder = SpanQueryBuilder(
         dataset=Dataset.Discover,
         params=params,
         selected_columns=selected_columns,
         query=query,
-        orderby=[direction + column for column in orderby_columns],
+        orderby=[],
         limit=per_suspect,
         offset=offset,
-        functions_acl=["array_join", "sumArray", "percentileArray", "maxArray"],
     )
 
+    # Make sure to resolve the custom span functions and add it to the columns and order bys
+    orderby_columns = [
+        builder.resolve_span_function(function, span, f"{function}_span_time")
+        for function in SPAN_PERFORMANCE_COLUMNS[orderby].suspect_example_functions
+    ]
+    builder.columns += orderby_columns
+    builder.orderby += [
+        OrderBy(column, Direction.DESC if direction == "-" else Direction.ASC)
+        for column in orderby_columns
+    ]
+
     # we are only interested in the specific op, group pairs from the suspects
     builder.add_conditions(
         [
-            Condition(
-                Function(
-                    "tuple",
-                    [
-                        builder.resolve_function("array_join(spans_op)"),
-                        builder.resolve_function("array_join(spans_group)"),
-                    ],
-                ),
-                Op.IN,
-                Function(
-                    "tuple",
-                    [Function("tuple", [span.op, span.group])],
-                ),
-            ),
+            Condition(Function("has", [builder.column("spans_op"), span.op]), Op.EQ, 1),
+            Condition(Function("has", [builder.column("spans_group"), span.group]), Op.EQ, 1),
```

Comment:
This is no longer properly using the bloom filter index on the span columns anymore.